File: src/main/java/top/zbeboy/isy/web/vo/internship/regulate/InternshipRegulateVo.java
package top.zbeboy.isy.web.vo.internship.regulate;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;
import java.sql.Date;

/**
 * Created by lenovo on 2016-12-24.
 */
@Data
public class InternshipRegulateVo {
    private String internshipRegulateId;
    @NotNull
    @Size(max = 10)
    private String studentName;
    @NotNull
    @Size(max = 20)
    private String studentNumber;
    @NotNull
    @Size(max = 15)
    private String studentTel;
    @NotNull
    @Size(max = 200)
    private String internshipContent;
    @NotNull
    @Size(max = 200)
    private String internshipProgress;
    @NotNull
    @Size(max = 20)
    private String reportWay;
    @NotNull
    private Date reportDate;
    private String schoolGuidanceTeacher;
    private String tliy = "无";
    @NotNull
    private Integer studentId;
    @NotNull
    @Size(max = 64)
    private String internshipReleaseId;
    @NotNull
    private Integer staffId;
}

File: src/main/java/top/zbeboy/isy/web/vo/internship/regulate/InternshipRegulateVo.kt
package top.zbeboy.isy.web.vo.internship.regulate

import java.sql.Date
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-29 .
 **/
open class InternshipRegulateVo {
    var internshipRegulateId: String? = null
    @NotNull
    @Size(max = 30)
    var studentName: String? = null
    @NotNull
    @Size(max = 20)
    var studentNumber: String? = null
    @NotNull
    @Size(max = 15)
    var studentTel: String? = null
    @NotNull
    @Size(max = 200)
    var internshipContent: String? = null
    @NotNull
    @Size(max = 200)
    var internshipProgress: String? = null
    @NotNull
    @Size(max = 20)
    var reportWay: String? = null
    @NotNull
    var reportDate: Date? = null
    var schoolGuidanceTeacher: String? = null
    var tliy = "无"
    @NotNull
    var studentId: Int? = null
    @NotNull
    @Size(max = 64)
    var internshipReleaseId: String? = null
    @NotNull
    var staffId: Int? = null
}